documentation/features.asciidoc

```diff
@@ -1,7 +1,7 @@
 :toc:
 toc::[]
 
-= Features
+== Why do you need devonfw-ide?
 
 Every developer needs great tools to work efficiently. Setting up these tools manually can be tedious and error-prone. Furthermore, some projects may require different versions and configurations of such tools. Especially configurations like code-formatters should be consistent within a project to avoid diff-wars.
```

Comment:
This is also in the introduction. Please avoid duplicate content and messages

Reply:
Removed features.asciidoc since the content already exists in introduction.